Branch to create Responsive Desktop Version of Portfolio

steveWDamesJr opened the pull request:
New branch created after encountering a "Resolve a merge conflict" error when I opened my pull request on the previous branch named "Portfolio-desktop-version". In order to fix error, that old branch had to be merged to main. After this I was able to continue with a New Branch and Pull request. 

# Here is a link to my previous Pull-Request: #
[This is the link of the old branch: "Portfolio-desktop-version"](https://github.com/steveWDamesJr/Microverse-Portfolio/pull/6 )
   

Refactor:
 -Markup to include desktop version elements of Figma Design: In Homepage, Works Section, About me and Contact Section 
-Stylesheet to reflect desktop version of Portfolio in above stated sections of Portfolio 
-Portfolio with media queries and changes to font, colors, sizing to reflect Figma design 
-All works cards, About me and Contact me section with restructuring for desktop view with flexbox and grid

Tests:
 -Linter tests to markup and stylesheet and Debug

steveWDamesJr committed:
Remove Conflicts from main and feature branch after

andersonlebon commented on index.html:
```diff
@@ -132,120 +170,135 @@ <h2 class="card-title">Tonic</h2>
+            <div class="frm-inputs-cont">
+              <label for="full-name"></label>
+              <input class="con-frm-input" type="text" name="name" id="full-name" placeholder="First and Last Name"
+                required maxlength="30">
+              <label for="email-address"></label>
+              <input class="con-frm-input" type="email" name="_replyto" id="email-address"
+                placeholder="email@microverse.org" required>
+              <label for="message"></label>
+              <textarea class="con-frm-mess" rows="5" name="message" id="message" placeholder="Write your message here"
+                required maxlength="500"></textarea>
+              <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
+              <input class="frm-button" type="submit" value="Get In Touch">
```
### Good job :clap: :clap: :+1:
 Your project is perfectly well designed and you really followed the same template's design however one of the requirements says that when designing your desktop version the mobile version should not be broken. 
- [x] KIndly your design would look better and sticks to The template by adding some `padding-top` on in the `textarea` in the form; check the image bellow :point_down:  and perform yours.
- [x] I would suggest aligning very well the `get in touch` button still in the form. check the image bellow :point_down: 

Your design | The template
---------------- | -----------------
![Screenshot from 2021-12-11 15-31-59](https://user-images.githubusercontent.com/65068771/145678967-a40bc5a6-b847-462e-a0f7-296437cbf749.png) | ![Screenshot from 2021-12-11 15-30-48](https://user-images.githubusercontent.com/65068771/145678970-11cc7754-a222-47c4-a190-17e07c328b10.png)

andersonlebon replied:
## [OPTIONAL]

- [x] Nice following button interactions but I think the design would look much better if your change the cursor to the pointer when hovering the button `get in touch ` in the form
- [x] The page title is adequately and briefly describes the content of the page `<title></title>`. For example:
- Poor titles:
 > Microverse project | Your portfolio.
 > Portfolio project
- Better page titles:
 > Mark's portfolio
 > Mark full-stack web developer

andersonlebon reviewed (changes requested):
Hi @steveWDamesJr 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

![almost-there](https://user-images.githubusercontent.com/65068771/141811112-252360b9-757f-4279-bb3b-b7a2a4675ad3.gif)
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your project is perctely well designed :clap:  :100: :+1: :heavy_check_mark: 
## Required Changes ♻️
- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

steveWDamesJr committed:
Add padding to contact me textarea placehelder

steveWDamesJr committed:
Fix Reviewer errors

-Add padding-top to textarea in Contact Me placeholder
-Update Get in Touch button
-Add pointer when hovering Get in Touch Button
-Update markup title element to more professional naming

steveWDamesJr committed:
Remove Conflicts from main and feature branch after commit

steveWDamesJr committed:
Remove Conflicts from main and feature branch after commit

steveWDamesJr committed:
Merge branch 'main' into Responsive-portfolio-desktop-version

steveWDamesJr committed:
Fix linter errors

steveWDamesJr committed:
Fix git conflict errors

steveWDamesJr committed:
Fix linter errors

steveWDamesJr committed:
Fix linter errors

gabrielyea reviewed (approved):
# Approved! :partying_face: :tada: 
Hi @steveWDamesJr 
Amazing work, you fixed all issues pointed out in the previous review!
<img src="https://media2.giphy.com/media/NJZMSqRY3rG9i/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
